src/_gettsim/parameters/sozialv_beitr.yaml

```diff
@@ -552,6 +552,14 @@ beitr_bemess_grenze_m:
       west: 7300
       ost: 7100
     reference: V. v. 28.11.2022 BGBl. I. S. 2128.
+  2024-01-01:
+    ges_krankv:
+      west: 5175
+      ost: 5175
+    ges_rentenv:
+      west: 7550
+      ost: 7450
+    reference: V. v. 24.11.2023 BGBl. 2023 I Nr. 322
 bezugsgröße_selbst_m:
   name:
     de: Monatliche Bezugsgröße
```

Comment:
This has been updated for 2024 as well: https://www.recht.bund.de/bgbl/1/2023/322/VO.html